cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -144,6 +147,17 @@ rules:
       - nodes
       - nodes/proxy
       - services
+      # Rook watches secrets for:
+      # the admission controller (TODO: can we remove this once the admission controller creates the self-signed cert?)
```

Comment:
rook watcher admission controller secret [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/webhook.go#L44) to create webhooks services, so I think this is required.
And, I think [here](https://github.com/rook/rook/blob/master/tests/scripts/deploy_admission_controller.sh#L107) self-signed cert is created only?

Reply:
Thanks for the info. :+1: